### PR TITLE
kernel-mshv: add back config

### PR DESCRIPTION
**Pull request by Redent0r**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?
We prefer to keep a copy of the config checked in for open source transparency

###### Change Log  

- kernel-mshv

###### Does this affect the toolchain?  

**NO**

###### Associated issues  

- #xxxx

###### Test Methodology

- Pipeline build id: https://dev.azure.com/mariner-org/mariner/_build/results?buildId=336254&view=results

### CODE REVIEW

File: SPECS/kernel-mshv/kernel-mshv.spec [hunk 1]
```diff
@@ -11,14 +11,15 @@
 Summary:        Mariner kernel that has MSHV Host support
 Name:           kernel-mshv
 Version:        5.15.98.mshv1
-Release:        2%{?dist}
+Release:        3%{?dist}
 License:        GPLv2
-URL:            https://microsoft.visualstudio.com/DefaultCollection/LSG/_git/linux-dom0
+URL:            https://github.com/microsoft/CBL-Mariner-Linux-Kernel
```
Comment (Redent0r) [inline SPECS/kernel-mshv/kernel-mshv.spec:16]: Wondering if this should be reverted back based on this [feedback](https://github.com/microsoft/CBL-Mariner/pull/5147#discussion_r1149292233)

Comment (ms-mahuber) [reply SPECS/kernel-mshv/kernel-mshv.spec:16]: CBL-Mariner-Linux-Kernel URL is definitely not right :) Let's follow up with the right people to give advice in such situations.

File: SPECS/kernel-mshv/kernel-mshv.spec [hunk 2]
```diff
@@ -11,14 +11,15 @@
 Summary:        Mariner kernel that has MSHV Host support
 Name:           kernel-mshv
 Version:        5.15.98.mshv1
```
Comment (ms-mahuber) [inline SPECS/kernel-mshv/kernel-mshv.spec:13]: in the kernel-uvm SPEC we're using "kernelver" as a variable but not here. any reason why we would not want to align this between both packages?

Comment (Redent0r) [reply SPECS/kernel-mshv/kernel-mshv.spec:13]: I'm more inclined to remove it from kernel-uvm SPEC. It just seems like unnecessary indirection, as you can already reference %{version}

**Review by ms-mahuber — APPROVED** [review]

**Pull request merged**